Code/Source/Camera/CameraSensor.cpp

```diff
@@ -30,6 +30,7 @@ namespace ROS2
         , cameraName(cameraName)
```

Comment:
with all class / struct member fields, follow m_<name> convention. It makes it easier to determine what is a part of state and what is an argument / local variable